src/components/Admin/UserPage.tsx

```diff
@@ -130,6 +130,7 @@ export const UserPage: FC<UserPageProps> = ({ mode = EditMode.readOnly, user, ti
         },
       });
     } else if (isEditMode && user.id) {
+      // eslint-disable-next-line @typescript-eslint/no-unused-vars
       const { email, ...userToUpdate } = userInput;
```

Comment:
The `email` extraction can be omitted. 
The back-end now checks if the `email` is different than the existing one, not only if it is in the input data. Please remove that and pass the `userInput` directly to the mutation.

Reply:
done